### PR TITLE
Make sure apt lists cleaned up in Dockerfile

### PR DESCRIPTION
**Pull request by PeterDaveHello**
Some of the Dockerfiles missed the part to clean up apt lists, this PR make sure every Dockerfile to do it!

### CODE REVIEW

**Comment by revant** [comment]
There seems to be some error with CI

File: build/frappe-worker/Dockerfile [hunk 1]
```diff
@@ -46,7 +46,8 @@ RUN apt-get update -y && apt-get install \
     && wget https://github.com/wkhtmltopdf/packaging/releases/download/0.12.6-1/wkhtmltox_0.12.6-1.buster_${ARCH}.deb \
     && dpkg -i wkhtmltox_0.12.6-1.buster_${ARCH}.deb && rm wkhtmltox_0.12.6-1.buster_${ARCH}.deb \
     && wget https://raw.githubusercontent.com/nvm-sh/nvm/v0.38.0/install.sh \
-    && chown -R frappe:frappe /home/frappe
+    && chown -R frappe:frappe /home/frappe \
+    && rm -rf /var/lib/apt/lists/* \
```
Comment (revant) [inline build/frappe-worker/Dockerfile:50]: does this trailing back slash (`\`) cause problem?

**Comment by revant** [comment]
I pushed a commit on your branch. It didn't trigger CI somehow. Closed this PR and reopened new one from your branch. Let's see what happens with CI now.

File: build/frappe-worker/Dockerfile (hunk 1 above)
Comment (PeterDaveHello) [reply build/frappe-worker/Dockerfile:50]: Sure

**Comment by PeterDaveHello** [comment]
@revant I can't reopen it, can you help?

**Comment by PeterDaveHello** [comment]
I've update the PR locally, should reopen this one and push again.

**Pull request merged**